src/components/Banner/Banner.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Banner = () => {
+	return (
+		<Wrapper>
+			Please note the trading fee has been temporarily increased to 2% (or 4% for long Synths to
+			short Synths){' '}
+			<Link
+				target="_blank"
+				href="https://blog.synthetix.io/planning-for-before-and-after-achernar/"
```

Comment:
would be great if this could use `ExternalLink` and the link put inside `constants/links`

Reply:
Hmmm Blogpost is going to be updated at some point. I probably wouldn't put it in a constant.